src/AbstractHyperrectangle.jl

```diff
@@ -59,6 +60,30 @@ function vertices_list(H::AbstractHyperrectangle{N}
         for si in Iterators.product([[1, -1] for i = 1:dim(H)]...)][:]
 end
 
+"""
+    constraints_list(H::AbstractHyperrectangle{N})::Vector{Vector{N}} where {N<:Real}
+
+Return the list of constraints of an axis-aligned hyperrectangular set.
+
+### Input
+
+- `H` -- hyperrectangular set
+
+### Output
+
+A list of linear constraints.
+"""
+function constraints_list(H::AbstractHyperrectangle{N})::Vector{LinearConstraint{N}} where {N<:Real}
+    n = dim(H)
+    constraints = Vector{LinearConstraint{N}}(2*n)
+    A, b, c = eye(n), high(H), -low(H)
```

Comment:
It is cheaper to create the row vectors of `A` individually. You could use `UnitVector`, see #538.